apps/agreement/contracts/Agreement.sol

```diff
@@ -289,15 +290,18 @@ contract Agreement is ILockManager, IAgreement, IArbitrable, IACLOracle, AragonA
     }
 
     /**
-    * @notice Sign the agreement
+    * @notice Sign the agreement up-to setting #`_settingId`
+    * @param _settingId Last setting ID the user is agreeing with, including all past settings
```

Comment:
I would say it's not "all past settings"; they're only signing for this particular version (the past ones don't matter).